Lib/multiprocessing/forkserver.py

```diff
@@ -25,6 +26,7 @@
 
 MAXFDS_TO_SEND = 256
 SIGNED_STRUCT = struct.Struct('q')     # large enough for pid_t
+_authkey_len = 32  # <= PIPEBUF so it fits a single write to an empty pipe.
```

Comment:
```suggestion
_AUTHKEY_LEN = 32  # <= PIPEBUF so it fits a single write to an empty pipe.
```

You may want to say in the NEWS entry that we use a 256-bit authentication key.

Reply:
that's an irrelevant to users internal implementation detail, no need to mention it.